cobigen/cobigen-tsplugin/src/main/java/com/devonfw/cobigen/tsplugin/merger/constants/Constants.java

```diff
@@ -21,6 +21,11 @@
     /** Export statement regex */
     public static final String EXPORT_REGEX = "export\\s+([^\\s]+)";
 
+    /**
+     * Path of the executable file where the server of the external process is stored
+     */
+    public static final String EXE_PATH = "src/test/resources/testdata/externalserver/nestserver.exe";
```

Comment:
you are accessing the test resources from the src main java. This will not work after packaging.
In addition, src/test/resources and src/main/java folders will vanish at build time and their contents will be placed on root.